mmv1/products/compute/OrganizationSecurityPolicy.yaml

```diff
@@ -18,6 +18,9 @@ base_url: 'locations/global/securityPolicies?parentId={{parent}}'
 self_link: 'locations/global/securityPolicies/{{policy_id}}'
 create_url: 'locations/global/securityPolicies?parentId={{parent}}'
 update_verb: :PATCH
+deprecation_message: >-
+  `google_compute_organizationsecuritypolicy` is deprecated and will be removed in the next major release
```

Comment:
I'm going to make this change to the docs as well - I think we should leave it open-ended which major release it will be removed in.

```suggestion
  `google_compute_organization_security_policy` is deprecated and will be removed in a future major release
```